Camera.MAUI/Platforms/Android/MauiCameraView.cs

```diff
@@ -177,9 +189,20 @@ internal async Task<CameraResult> StartRecordingAsync(string file, Microsoft.Mau
                         mediaRecorder.SetVideoEncoder(VideoEncoder.H264);
                         mediaRecorder.SetAudioEncoder(AudioEncoder.Aac);
                         IWindowManager windowManager = context.GetSystemService(Context.WindowService).JavaCast<IWindowManager>();
+
+                        // Below is making the camera upside down.....in recording...
```

Comment:
This comment should be removed (or reworded). The upside-down issue is solved by now. The code below is fixing the orientation, not breaking it.

Reply:
All set apologize for the delay, had some other pressing matters.